src/helpers/get-os-language.ts

```diff
@@ -0,0 +1,25 @@
+import { I18nManager, Platform, Settings } from "react-native";
+
+function getOSLanguage(): string | undefined {
```

Comment:
could we rename the file to `getOsLanguage` instead of `get-os-language` to keep the camelCase pattern?